xversionmessage.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+#include "xversionmessage.h"
+//#include "hashwrapper.h"
+//#include "random.h"
+//#include "streams.h"
+//#include "util.h"
+
```

Comment:
to be remove?

Reply:
I want to "additively remove" everything from this file to make it easy to grab new versions.

Reply:
I seed